fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTabletStorageFormatStmt.java

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.UserException;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+public class ShowTabletStorageFormatStmt extends ShowStmt {
+    private boolean verbose;
+
+    @Override
+    public boolean isVerbose() {
+        return verbose;
+    }
+
+    public ShowTabletStorageFormatStmt(boolean verbose) {
+        this.verbose = verbose;
+    }
+
+    @Override
+    public void analyze(Analyzer analyzer) throws UserException {
+        super.analyze(analyzer);
```

Comment:
check privilege here, and I think it should be ADMIN priv.
And the stmt is better to be:
`ADMIN SHOW TABLET STORAGE FORMAT`